contents/handbook/growth/sales/contract-rules.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Contract Rules
+sidebar: Handbook
+showTitle: true
+---
+
+We get it, nobody likes rules.  As we scale the Sales and CS teams we need to be consistent in how we approach contracts to ensure that we are setting ourselves up for future success.  If you have a customer situation where you'd like to vary the rules below, get approval from Dana, Simon or Charles first.  You should also have them note the approval on the Salesforce opportunity record.
+
+## Discounts
+
+> We don't offer discounts to customers paying monthly with no commitment aside from non-profit orgs (see below).  You might see some 
+> customers who have a 30% B2C discount.  This is a relic from when all of our pricing was bundled and we should look to migrate them to our standard pricing if 
+> at all possible.
+
+Although our standard monthly [pricing](/pricing) has volume discounts built in, it's common practise when negotiating software contracts for the customer (and their procurement team) to ask for a discount.  We can be super transparent about the levers we have in discounting PostHog:
+
+1. In our consumption-based pricing model, the easiest way to reduce spend is to ensure that the customer is only sending data to us which is valuable to them.  There is [different guidance here](/docs/billing/estimating-usage-costs) depending on which product(s) they are looking at.
+2. **Annual Plan:** We offer a 20% discount on an annual paid-up-front plan, because it's beneficial to PostHog to have a customer committed for a year.
+3. **Higher Spend:** We offer an additional 5% and then 15% on top of the annual plans when the customer spends above $60k and $100k respectively.
+4. **Multi-year:** We offer an additional 5% for each additional year of commitment, as longer commitments are beneficial to PostHog (see table below). Credits rollover in-between for multi-year contracts.
+5. **Up-front payment:** We offer an additional 3% where a *multi-year* contract is all **paid up-front**, as it's better for PostHog to have the money in the bank.
+
+> We don't offer a discount for signing a contract by some fixed date (e.g so the deal closes in a specific quarter).
+
+> If you need to vary these rules to sign a customer up speak to Dana, Simon or Charles about it before quoting an additional discount to the customer. We don't want to put a rigorous approval process in place here but equally these rules are in place for a legitimate business reason. If you go outside of these rules without clearing it with one of us first, the deal may not count towards your quota.
```

Comment:
Can we harden the language here a bit so customers don't see it and go 'yeah go on give us a discount'

Reply:
> You shouldn't offer discounts above the levels outlined here.  If a customer is asking for more and you feel like it is warranted (not just for an easy win) then speak to Dana, Simon or Charles about it.  We may be able to offer additional credit for the first year of a contract but we shouldn't do this as a default.  If you go outside of these rules without clearing it with one of us, the deal may not count toward your quota.

?

Reply:
Slight tweak:

You shouldn't offer discounts above the levels outlined here. If a customer is asking for more and you feel like a genuine exception is justified then speak to Dana, Simon or Charles about it as we _may_ be able to offer additional credit for the first year of a contract. If you go outside of these rules without clearing it with one of us, the deal may not count toward your quota.